src/v2/Apps/Auction/Components/__tests__/AuctionActiveBids.jest.tsx

```diff
@@ -109,6 +109,36 @@ describe("AuctionActiveBids", () => {
       expect(wrapper.text()).toContain("$2000")
     })
 
+    describe("auction with cascading lots, some closed", () => {
+      it("displays bidding closed for a lot that has closed", () => {
+        const wrapper = getWrapper({
+          Me: () => ({
+            lotStandings: [
+              {
+                saleArtwork: {
+                  counts: {
+                    bidderPositions: 1,
+                  },
+                  endedAt: "2021-01-01T00:00:00.000Z",
+                },
+              },
+              {
+                saleArtwork: {
+                  counts: {
+                    bidderPositions: 2,
+                  },
+                  endedAt: null,
+                },
+              },
+            ],
+          }),
+        })
+
+        expect(wrapper.text()).toContain("Bidding Closed")
+        expect(wrapper.text()).toContain("Increase Bid")
```

Comment:
a single view is supposed to have Bidding closed and increase bid?

Reply:
It has 2 lots in the view.